### PR TITLE
Update locales-bg-BG.xml

### PR DESCRIPTION
**Pull request by valeriina**
all fields translated
some fields corrected
deleted some forms for single and plural case - where it needed

### CODE REVIEW

File: locales-bg-BG.xml [hunk 1]
```diff
@@ -204,22 +228,16 @@
     </term>
 
     <!-- LONG ROLE FORMS -->
-    <term name="director">
-      <single>director</single>
-      <multiple>directors</multiple>
-    </term>
+    <term name="director">под общата редакция на</term>
```
Comment (adam3smith) [inline locales-bg-BG.xml]: I think this is a misunderstanding of the term -- this director as in movie director. Shouldn't that be режисьор ?

**Comment by adam3smith** [comment]
@valeriina sorry this took forever to review. In general looks great, but I have a question about the translation of "director" -- would you mind taking a look?

**Comment by valeriina** [comment]
Hello everyone,
If the template is for books, the given term is correct (под общата редакция на), if is about art and movies then the appropriate term is режисьор.

All the best!
Valeeiya Simeonova

**Comment by adam3smith** [comment]
Thanks! And apologies for the delay

**Pull request merged**